addons/crm/models/crm_lead.py

```diff
@@ -1231,7 +1231,8 @@ def get_empty_list_help(self, help):
             help_title = _('Create a new lead')
         else:
             help_title = _('Create an opportunity to start playing with your pipeline.')
-        alias_record = self.env['mail.alias'].search([
+        sales_team = self.env.user.sale_team_id
+        alias_record = sales_team.alias_id if sales_team.active else self.env['mail.alias'].search([
```

Comment:
To avoid multi-company weirdness where the relation shouldn't be accessible in the current context, let's do:
```py
    sales_team_domain = [('member_ids', 'in', self.env.user.id)] if self.env.user.sale_team_id.active else []
    self.env['crm.team'].search(['alias_id', '!=', False] + sales_team_domain, limit=1).alias_id
```

Reply:
Do we need to check for alias_id being defined since it's a required field?

Reply:
I think the intention is to only search crm.team with alias (there could be a crm.team without alias that would not be useful to show in the "you can send a mail to this team"

Reply:
Right, my bad, you should just check alias_name as is done below. The intention is indeed simply to avoid those without actual alias